jerr/const.go

```diff
@@ -18,62 +18,63 @@ const (
 	ProtocolNotFound     = `the directive "Protocol" not found`
 	ProtocolParameterErr = `the parameter value have to be "json-rpc-2.0"`
 
-	InfoIsEmpty        = "info is empty"
-	ResponsesIsEmpty   = "responses is empty"
-	RequestIsEmpty     = "request is empty"
-	DescriptionIsEmpty = "empty is description"
-	BodyIsEmpty        = "empty is body"
-	MacroIsEmpty       = "macro is empty"
+	InfoIsEmpty        = "the INFO directive cannot be empty"
+	ResponsesIsEmpty   = "the response cannot be empty"
+	RequestIsEmpty     = "the request cannot be empty"
+	DescriptionIsEmpty = "the description cannot be empty, learn more about the Description directive here: https://jsight.io/docs/jsight-api-0-3#directive-description"
+	BodyIsEmpty        = "the body cannot be empty"
+	MacroIsEmpty       = "the macros cannot be empty, learn more about the MACRO directive here: https://jsight.io/docs/jsight-api-0-3#directive-macro"
 
 	IncorrectPath             = "incorrect path"
 	IncorrectRequest          = "incorrect request"
-	IncorrectDirectiveContext = "incorrect directive context"
+	IncorrectDirectiveContext = "incorrect context for the directive"
 	IncorrectParameter        = "incorrect parameter"
 
-	PathOrErr                          = "the root schema object cannot have an OR rule in the Path directive"
-	PathObjectErr                      = "the body of the Path DIRECTIVE must be an object"
-	PathAdditionalPropertiesErr        = `the "additionalProperties" rule is invalid in the Path directive`
-	PathNullableErr                    = `the "nullable" rule is invalid in the Path directive`
-	PathEmptyErr                       = "an empty object in the Path directive"
+	PathOrErr                          = "the root schema object cannot have the `or` rule in the Path directive"
+	PathObjectErr                      = "the body of the Path directive must be an object, learn more about the Path directive here: https://jsight.io/docs/jsight-api-0-3#directive-path"
+	PathAdditionalPropertiesErr        = `the "additionalProperties" rule is invalid in the Path directive, learn more about the Path directive here: https://jsight.io/docs/jsight-api-0-3#directive-path`
```

Comment:
'invalid' may be misleading, it may be  interpreted, that it is generally allowed, but somehow invalid in this particular case, like a syntax error.

Reply:
fixed!